### DISCUSSION TITLE
[ 灌水 ] 莫妮卡大姐姐的新单曲你们听了吗？

### DISCUSSION THREAD

**Discussion by wugan-bot156** [share121/inter-knot#403] [其他]
![屏幕截图 2024-08-10 021534](https://github.com/user-attachments/assets/f9a7b878-8db3-44a7-86a9-42625a8acad2)
莫妮卡大姐姐的新单曲MV你们看了吗?
感觉好看里有一丝难听，难听里又透露着好看。
眼睛很享受，耳朵很痛苦。

**Comment by WORLD-executeME** [comment]
我同意，MV很好看但这首单曲本身...很难评，只能说我实在欣赏不来这首曲子

**Comment by Ailian-Qiao** [comment]
听了，说是在回馈粉丝，感觉更像是在报复粉丝...

**Comment by kano0x0** [comment]
听说六分街录像店的店主很会鉴赏音乐，我可以帮您联系他

**Reply by nahida0-o** [reply]
这个帖子是在模仿绝区零游戏里的一个帖子，你可以去游戏里的绳网找找看，真的有。
![Uploading aadc4458521ca34ec07b093977de501.jpg…]()

**Comment by nahida0-o** [comment]
这张单曲是经典爵士老歌。我记得有个叫【女王】的地下偶像翻唱过，有免费的音
源。
比莫妮卡唱得好听多了！还不用花钱听！

**Comment by banyue121** [comment]
是《噢~甜心》那个莫妮卡...吗?

**Comment by kano0x0** [comment]
这张单曲是经典爵士老歌。我记得有一个叫⌈ 女王 ⌋的地下偶像翻唱过，有免费的音源。
比莫妮卡唱得好听多了！还不用花钱听！
